contracts/SecurityTokenRegistry.sol

```diff
@@ -90,6 +91,8 @@ contract SecurityTokenRegistry is EternalStorage, Proxy {
     event ChangeSecurityLaunchFee(uint256 _oldFee, uint256 _newFee);
     // Emit when changeTickerRegistrationFee is called
     event ChangeTickerRegistrationFee(uint256 _oldFee, uint256 _newFee);
+    // Emit when Fee currency is changed
+    event ChangeFeeCurrency(bool _oldIsFeesInPoly, bool _newIsFeesInPoly);
```

Comment:
`is` should be prefixed in the param names.

Reply:
I decided to remove `_oldIsFeesInPoly` as it is always gonna be the opposite of new fees. renamed the new fees var.